src/lib/y2network/dialogs/edit_interface.rb

```diff
@@ -0,0 +1,58 @@
+require "cwm/dialog"
+
+require "y2network/widgets/address_tab.rb"
+require "y2network/widgets/bond_slaves_tab.rb"
+require "y2network/widgets/bridge_slaves_tab.rb"
+require "y2network/widgets/general_tab.rb"
+require "y2network/widgets/hardware_tab.rb"
+
+Yast.import "LanItems"
+Yast.import "Label"
+
+module Y2Network
+  module Dialogs
+    # Dialog to create or edit route.
+    class EditInterface < CWM::Dialog
```

Comment:
In general, would be nice to have new classes proper documented. In this case, at least the `settings` param.